frame/composable-traits/src/dex.rs

```diff
@@ -178,6 +178,9 @@ pub trait Amm {
 		min_receive: Self::Balance,
 		keep_alive: bool,
 	) -> Result<Self::Balance, DispatchError>;
+
+	/// Get current price of LP token.
+	fn get_price_of_lp_token(pool_id: Self::PoolId) -> Result<Self::Balance, DispatchError>;
```

Comment:
Shouldn't we add this function to the `runtime` (`dali`), calculate weights for it?

P.S. The same goes for `update_accounts_deposited_one_asset_storage` function actually.

Reply:
To be honest, I don't know.

Reply:
Only extrinsics need weights, as far as i can tell this isn't called as an extrinsic

Reply:
Then be free to close (resolve) the thread if this function does not plan to be an extrinsic.